projects/core/utils/format/format-phone.ts

```diff
@@ -1,6 +1,16 @@
 import {CHAR_PLUS} from '@taiga-ui/cdk';
 
 /**
+ * TODO: delete in v4.0
+ * @deprecated Use {@link https://tinkoff.github.io/maskito/core-concepts/transformer maskitoTransform} instead<br/>
+ * @example
+ * import {maskitoTransform} from '@maskito/core';
+ *
+ * const mask = {
+ *     mask: ['+', '1', ' ', /\d/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/],
```

Comment:
I wonder if we need a public API instrument to turn ##-### mask representation to `MaskitoOptions` 🤔